lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -94,7 +94,7 @@ export class ManagedIdentityApplication {
             ManagedIdentityApplication.nodeStorage as NodeStorage,
             fakeAuthorityOptions,
             this.logger,
-            undefined,
+            this.config.managedIdentityId.id,
```

Comment:
needed for changes made to msal-common for a bugfix

Reply:
What is the bug fix?

Reply:
In Authority.ts, the correlationId's type changed from `undefined | string`, to just `string`

Reply:
That's not a correlation ID, that's a managed identity id. Can you please let it null?

Reply:
Correct. This is for the fake Authority, which is used to create a fake ClientCredentialClient... which is needed to reuse cache functionality. Any string value can be passed in.

Reply:
Every request should have a correlation ID, even if (for now) we can't inject the correlation ID into MSI. My 2c - just generate a GUID for each request. And ensure each log statement logs it.